CHANGES.rst

```diff
@@ -11,6 +11,12 @@ datamodels
 
 - Update keyword comments/titles for V2_REF, V3_REF, FITXOFFS, FITYOFFS [#6822]
 
+ramp_fitting
```

Comment:
Should also add entries for other steps affected, such as linearity, refpix, saturation, and superbias.

Reply:
I added those entries.